nixos/modules/services/monitoring/zabbix-proxy.nix

```diff
@@ -0,0 +1,133 @@
+# Zabbix proxy daemon.
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.zabbixProxy;
+
+  stateDir = "/run/zabbix";
+
+  logDir = "/var/log/zabbix";
+
+  libDir = "/var/lib/zabbix";
+
+  pidFile = "${stateDir}/zabbix_proxy.pid";
+
+  configFile = pkgs.writeText "zabbix_proxy.conf"
+    ''
+      Server = ${cfg.server}
+
+      LogFile = ${logDir}/zabbix_proxy
```

Comment:
Logging to the system log is preferred if possible. If not possible does `zabbix` take care of its own log rotation?

Reply:
Sadly no systemd integration here. We will use `logrotate`.